docs/getting-started.md

```diff
@@ -0,0 +1,71 @@
+# Getting started
+
+You will need to be running the version of NodeJS set in the `.nvmrc` file in the root of the repository and `Yarn` package manager.
+
+You can quickly install Yarn by running `npm i -g yarn`. If you are running multiple versions of Node on your machine you will need to make sure you install Yarn on the specific version Node you are using as the installation will not jump between different versions of NodeJS.
+
+## Installation
+
```

Comment:
One thing that confused me is that this is the installation section but in the section above you install yarn globally. Would it make sense to have the yarn install in this section?

Reply:
this repo is also using npm since it's package-lock.json (yarn's is package.lock) so be good to clear this up. (I assume it's because digital-form-builder requires yarn, so if that's the case that's ok to just mention as well)

Reply:
So the installation section was more for the project dependencies (i.e. running `npm ci` and the form runner install script). Yarn is kind of a top level dependency/requirement alongside Node which is why it's placed up there at the start alongside the text about needing to have NodeJS installed.

Reply:
Ah, I get what you mean now (ignore my previous comment). I've updated this to explain why you need both `npm` and `yarn` (because the form runner needs Yarn)